README.md

```diff
@@ -3,11 +3,15 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
 ## Getting Started
 
 
-First, run the development server:
+First, install dependencies:
+
+```bash
+yarn ci
```

Comment:
I don't think yarn has `yarn ci` command. I can't find `yarn ci` command on documentation. We can just do `yarn install` or just run `yarn` for installing dependencies. https://classic.yarnpkg.com/en/docs/cli/install